package.json

```diff
@@ -21,6 +21,7 @@
     "@mui/x-charts": "^7.22.2",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-icons": "^5.3.0",
```

Comment:
There is already the library @mui/icons-material, it would be better to use just one as the project size is increased with more libraries, and react-icons offers different libraries of icons which can cause inconsistency.